datarepo-clienttests/src/main/java/common/commands/LockAndRunTest.java

```diff
@@ -0,0 +1,52 @@
+package common.commands;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import runner.TestRunner;
+import runner.config.ServerSpecification;
+import runner.config.TestConfiguration;
+
+public class LockAndRunTest {
+  private static final Logger logger = LoggerFactory.getLogger(LockAndRunTest.class);
+
+  public static void main(String[] args) throws Exception {
+    if (args.length != 2) { // if no args specified or invalid number of args specified, print help
+      PrintHelp.printHelp();
+      return;
+    }
+
+    LockNamespace.lockNamespace();
+    unlockShutdownHook();
+
+    // execute a test configuration or suite
+    boolean isFailure;
+    isFailure = TestRunner.runTest(args[0], args[1]);
+    if (isFailure) {
+      System.exit(1);
+    }
```

Comment:
Do you need a call to UnlockNamespace.unlockNamespace() after running the test, or does the shutdown hook take care of that?

Reply:
The shutdown hook should take care that! I think I removed it.